### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by krofax**

### CODE REVIEW

File: docs/api/js/getting-started.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ::: warning
 - Please note that with the system update released in Feb 2023, the `ergs` concept is only used by the VM while the API layer operates with `gas`.
-- For more information, read the [changelog](../troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
+- For more information, read the [changelog](../../dev/troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
```
Comment (kailash-manasarovar) [inline docs/api/js/getting-started.md:5]: I don't think this is broken.

File: docs/dev/developer-guides/bridging/l1-l2.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This section describes the interface for interaction with zkSync from L1. It ass
 
 ::: warning
 - Please note that with the system update released in Feb 2023, the `ergs` concept is only used by the VM while the API layer operates with `gas`.
-- For more information, read the [changelog](../troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
+- For more information, read the [changelog](../../troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
```
Comment (kailash-manasarovar) [inline docs/dev/developer-guides/bridging/l1-l2.md:7]: I don't think this is broken either.

File: docs/dev/developer-guides/bridging/l2-l1.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ This section describes the interface to interact with Ethereum from L2. It assum
 
 ::: warning
 - Please note that with the system update released in Feb 2023, the `ergs` concept is only used by the VM while the API layer operates with `gas`.
-- For more information, read the [changelog](../troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
+- For more information, read the [changelog](../../troubleshooting/changelog.md#hardhat-plugins-update-feb-24th-2023).
```
Comment (kailash-manasarovar) [inline docs/dev/developer-guides/bridging/l2-l1.md:8]: I don't think this is broken.

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
The fix is of three links on the page recorded in the first column in the pic in the ticket.

**Comment by krofax** [comment]
> The fix is of three links on the page recorded in the first column in the pic in the ticket.

There were broken, found out when I ran `yarn lint:dead`

**Comment by kailash-manasarovar** [comment]
> 

- Yes, there are three broken links that you fixed by creating dummy pages for. That is not necessary.
- The changelog links you fixed aren't broken, for example: https://era.zksync.io/docs/api/js/getting-started.html try it.

**Comment by krofax** [comment]
> > 
> 
> * Yes, there are three broken links that you fixed by creating dummy pages for. That is not necessary.
> * The changelog links you fixed aren't broken, for example: https://era.zksync.io/docs/api/js/getting-started.html try it.

- Totally agree the pages ain't necessary, I did a redirect instead(happy to know your thoughts on this). However, when we do have all the SDK revamped we should add a `README.md` file in them just like the others.

- It doesn't look broken on prod, but it does on local, it just makes sense to link properly, the `ENV`(local, staging, or prod notwithstanding). yeah?

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
This is also unnecessary. The broken links are on this page: https://era.zksync.io/docs/dev/ just fix the links on this page.

**Comment by kailash-manasarovar** [comment]
> > > 
> > 
> > 
> > 
> > * Yes, there are three broken links that you fixed by creating dummy pages for. That is not necessary.
> > * The changelog links you fixed aren't broken, for example: https://era.zksync.io/docs/api/js/getting-started.html try it.
> 
> * Totally agree the pages ain't necessary, I did a redirect instead(happy to know your thoughts on this). However, when we do have all the SDK revamped we should add a `README.md` file in them just like the others.

I'm not convinced a README page with nothing on it but a manual toc which we have to remember to keep up to date is a good idea.

> * It doesn't look broken on prod, but it does on local, it just makes sense to link properly, the `ENV`(local, staging, or prod notwithstanding). yeah?

I don't have the issue locally. Have you got the latest updates?

File: docs/dev/README.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ token.
 - [Hardhat Plugins](../api/hardhat/) - Use our Hardhat zkSync plugins to compile, test, deploy, and verify your Solidity or Vyper based 
 - [zkSync CLI](../api/tools/zksync-cli/) - Simplify your development process and interact with zkSync Era from your terminal with zkSync CLI.
 - [Compiler Toolchain](../api/compiler-toolchain/overview.md) - Learn about our compiler toolchain.
-- [Python SDK](../api/python/) - Explore all the Python methods and functions required to build on zkSync Era.
-- [Go SDK](../api/go/) - Explore all the GO methods and functions required to build on zkSync Era.
-- [Java SDK](../api/java/) - Explore all the Java methods and functions required to build on zkSync Era.
-- [Swift SDK](../api/swift/) - Explore all the Swift methods and functions required to build on zkSync Era.applications.
\ No newline at end of file
+- [Python SDK](../api/python/getting-started.md) - Explore all the Python methods and functions required to build on zkSync Era.
```
Comment (kailash-manasarovar) [inline docs/dev/README.md:61]: not this one

**Review by kailash-manasarovar — APPROVED** [review]
LGTM

**Pull request merged**